moabb/datasets/preprocessing.py

```diff
@@ -89,6 +118,9 @@ def transform(self, raw, y=None):
             log.warning("No events found, skipping setting annotations.")
         return raw
 
+    def __repr__(self):
+        return f"SetRawAnnotations({self.event_id}, {self.interval})"
+
```

Comment:
Why do you need this?
I already get exactly this representation when I `print(paradigm.make_process_pipelines())`

Reply:
because the visualisation at the visual block needs this __repr__ at least on jupyter notebook.

Reply:
But it's already implemented in `BaseEstimator`:
```python
In [4]: RawToEvents(event_id={"a":1}, interval=[0,1]).__repr__()
Out[4]: "RawToEvents(event_id={'a': 1}, interval=[0, 1])"

In [5]: RawToEvents(event_id={"a":1}, interval=[0,1]).__repr__
Out[5]: <bound method BaseEstimator.__repr__ of RawToEvents(event_id={'a': 1}, interval=[0, 1])>
```

Reply:
oh. my bad